test/langtools/tools/javap/T6716452.java

```diff
@@ -70,7 +70,7 @@ <T extends Attribute<T>> void test(MethodModel mm, AttributeMapper<T> attr, Clas
                 if (!c.isAssignableFrom(mm.attributes().get(index).getClass())) {
                     error(mm + ": unexpected attribute found,"
                             + " expected " + c.getName()
-                            + " found " + mm.attributes().get(index).attributeName());
```

Comment:
Is this one redundant?

Reply:
Unfortunately no, toString includes Utf8Entry info.